backward.hpp

```diff
@@ -1325,7 +1325,7 @@ class TraceResolverLinuxImpl<trace_resolver_tag::libdw>:
 	ResolvedTrace resolve(ResolvedTrace trace) {
 		using namespace details;
 
-		Dwarf_Addr trace_addr = (Dwarf_Addr) trace.addr;
+		Dwarf_Addr trace_addr = reinterpret_cast<uintptr_t>(trace.addr);
```

Comment:
Shouldn't be `reinterpret_cast<Dwarf_Addr>`?

Reply:
First I  did not realize that I pushed a similar pull request fixing different warnings. For this particular error, I fixed differently. I may wrong but trace.addr is void* and Dwarf_Addr is long unsigned int. So I used https://stackoverflow.com/questions/39319862/convert-void-to-int-without-warning